easybib/spec/easybib_deploy_manager_spec.rb

```diff
@@ -12,7 +12,7 @@
   let(:runner) do
     ChefSpec::Runner.new(
       :cookbook_path => cookbook_paths,
-      :log_level => :debug,
+      :log_level => :error,
```

Comment:
you could also add 
```
:platform => 'ubuntu',
:version => '14.04'
```

and require chefspec instead of the spec helper - this way, the test is more decoupled from the rest. What do you think?